app.py

```diff
@@ -15,9 +15,115 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""This is the main script of the template project."""
+"""Graph metrics exporter logic for the Thoth project."""
+
+import os
+import logging
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+from thoth.common import init_logging
+from thoth.storages import GraphDatabase
+from thoth.storages import __version__ as __storages_version__
+from thoth.common import __version__ as __common_version__
+
+
+__version__ = "0.0.1"
+__service_version__ = f"{__version__}+" f"common.{__common_version__}.storages.{__storages_version__}"
+
+_LOGGER = logging.getLogger("thoth.graph_metrics_exporter")
+
+COMPONENT_NAME = "graph-metrics-exporter"
+
+# database
+KNOWLEDGE_GRAPH_HOST = os.getenv("KNOWLEDGE_GRAPH_HOST", "localhost")
+KNOWLEDGE_GRAPH_PORT = os.getenv("KNOWLEDGE_GRAPH_PORT", "5432")
+KNOWLEDGE_GRAPH_USER = os.getenv("KNOWLEDGE_GRAPH_USER", "postgres")
+KNOWLEDGE_GRAPH_DATABASE = os.getenv("KNOWLEDGE_GRAPH_DATABASE", "postgres")
+
+# metrics
+PROMETHEUS_REGISTRY = CollectorRegistry()
+THOTH_METRICS_PUSHGATEWAY_URL = os.environ["PROMETHEUS_PUSHGATEWAY_URL"]
+THOTH_DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
+
+GRAPH_METRICS_EXPORTER_TASK = os.environ["THOTH_GRAPH_METRICS_EXPORTER_TASK"]
+
+REGISTERED_TASKS = ["graph_corruption_check"]
+
+init_logging()
+
+database_schema_revision_script = Gauge(
+    "thoth_database_schema_revision_script",
+    "Thoth database schema revision from script",
+    ["component", "revision", "env"],
+    registry=PROMETHEUS_REGISTRY,
+)
+
+graphdb_is_corrupted = Gauge(
+    "thoth_graphdb_is_corrupted",
+    "amcheck has detected corruption.",
+    [],
+    registry=PROMETHEUS_REGISTRY,
+)
+
+
+def _create_common_metrics(component_name: str = COMPONENT_NAME):
+    """Create common metrics to pushgateway."""
+    deployment_name = THOTH_DEPLOYMENT_NAME
+
+    if deployment_name:
```

Comment:
`THOTH_DEPLOYMENT_NAME` is declared on the global scope - no reason to have it assigned above to a local variable. Also the check here is misleading. As you access `THOTH_DEPLOYMENT_NAME` variable directly using `os.environ["THOTH_DEPLOYMENT_NAME"]`, it will be always set (otherwise app.py crashes on startup with an appropriate exception). The check here checks whether this variable is empty. For simplicity, I think this can be removed.